assets/sass/components/user-input/googlekitekit-user-input-module-header.scss

```diff
@@ -0,0 +1,267 @@
+				margin-left: -10px;
+				margin-right: -10px;
+			}
+		}
+
+		&__next-question::before {
+			background-color: $c-white;
+			content: "";
+			height: 100%;
+			left: 0;
+			opacity: 0.75;
+			position: absolute;
+			top: 0;
+			width: 100%;
+			z-index: 10;
+		}
+
+		&__question-number {
+			color: $c-silver;
+			font-size: 36px;
+			line-height: 44px;
+			margin: 0;
+
+			@media (min-width: $bp-tablet) {
+				font-size: 28px;
+				line-height: 36px;
+			}
+		}
+
+		&__question-instructions {
+
+			& p {
+				color: $c-black;
+				font-size: 22px;
+				line-height: 28px;
+
+				@media (min-width: $bp-tablet) {
+					font-size: 14px;
+					line-height: 20px;
+				}
+			}
+
+			& h1 {
+				color: $c-black;
+				font-size: 36px;
+				line-height: 44px;
+				margin: 13px 0;
+
+				@media (min-width: $bp-tablet) {
+					font-size: 28px;
+					line-height: 36px;
+				}
+			}
+
+			&--notice {
+				background-color: rgba($c-text-notice-info, 0.2);
+				display: inline-block;
+				margin-bottom: 24px;
+				padding: 6px 14px;
+			}
+
+			&--author {
+				align-items: center;
+				color: $c-scarpa-flow;
+				display: flex;
+				font-size: 12px;
+
+				& img {
+					border-radius: 50%;
+					height: 24px;
+					margin-right: 8px;
+					width: 24px;
+				}
+			}
+		}
+
+		&__select-option {
+			align-items: center;
+			border-bottom: 1px solid $c-mercury;
+			display: flex;
+			padding: 15px;
+
+			& .mdc-text-field {
+				background-color: transparent;
+				height: 36px;
+				margin-left: 1rem;
+
+				input {
+					border: none;
+					padding: 0;
+				}
+			}
+		}
+
+		&__text-options {
+			border: 2px solid $c-jumbo;
+			border-radius: 5px;
+			display: flex;
+			margin-top: 13px;
+			padding: 8px;
+
+			& .mdc-text-field {
+				background-color: transparent;
+				height: 40px;
+
+				& input {
+					border: none;
+					padding: 0;
+				}
+			}
+		}
+
+		&__text-option,
+		&__preview-answer {
+			align-items: center;
+			background-color: $c-solitude;
+			border-radius: 100px;
+			color: $c-denim;
+			display: flex;
+			font-weight: 500;
+			margin-right: 12px;
+			padding: 3px 3px 3px 12px;
+			white-space: nowrap;
+		}
+
+		&__text-option button {
+			border-radius: 100px;
+			margin: 0;
+			min-width: 0;
+			padding: 0 12px;
+		}
+
+		&__preview-answers {
+			display: flex;
+			flex-wrap: wrap;
+		}
+
+		&__preview-answer {
+			margin-bottom: 12px;
+			padding-right: 12px;
+		}
+
+		&__preview-group {
+			border-bottom: 1px solid $c-mercury;
+			margin-bottom: 12px;
+			padding-bottom: 12px;
+		}
+
+		&__preview-group-title {
+			align-items: flex-start;
+			display: flex;
+			justify-content: space-between;
+
+			& p {
+				color: $c-cape-cod;
+				margin-top: 0;
+			}
+
+			& button {
+				font-weight: 400;
+				margin-top: -0.5em;
+				text-transform: none;
+			}
+		}
+
+		&__congrats {
+			background-color: $c-white;
+			margin: -10px;
+			padding: 35px;
+
+			@include shadow;
+
+			& h1 {
+				margin-bottom: 0;
+			}
+		}
+	}
+}
```

Comment:
Great work on already adding all the CSS here - although most of that is technically part of #2040. But that clearly reduces the necessary work there, we'll probably only need to make some minor UI refinements there which I won't address in here.